src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,17 @@
 import React from 'react';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../type/Movie';
+
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+interface Movies {
+  movies: Movie[],
+}
+
+export const MoviesList: React.FC<Movies> = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <MovieCard {...movie} />
```

Comment:
Pass the object as a whole, or as separate properties.
https://mateacademy.slack.com/archives/C02CSGP1KD1/p1636094929188700